tests/kvlds-blocking/test_kvlds_blocking.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Paths
+LBS=../../lbs/lbs
+KVLDS=../../kvlds/kvlds
+TESTKVLDS=./test_kvlds_blocking
+STOR=${KIVALOO_TESTDIR:-`pwd`/stor}
+SOCKL=$STOR/sock_lbs
+SOCKK=$STOR/sock_kvlds
+
+# Clean up any old tests
+rm -rf $STOR
+
+# Start LBS
+mkdir $STOR
+[ `uname` = "FreeBSD" ] && chflags nodump $STOR
+$LBS -s $SOCKL -d $STOR -b 512 -l 1000000
```

Comment:
If you launch lbs and kvlds with `-1` then we don't need to kill them at the end.